README.md

```diff
@@ -29,104 +30,158 @@ applications. More information about Thread can be found on
 
-Nest, along with ARM, Atmel, a subsidiary of Microchip Technology,
-Dialog Semiconductor, Microsoft Corporation, Nordic Semiconductor,
-Qualcomm Technologies, Inc. (a subsidiary of Qualcomm Incorporated),
-and Texas Instruments Incorporated are contributing to the ongoing
-development of OpenThread.
+1. Learn more about the [OpenThread architecture and features](#openthread-features)
+1. Get familiar with [platforms and devices that support OpenThread](#who-supports-openthread)
+1. See what [testing tools](#what-tools-are-available-for-testing) are available
+1. Learn [where to get help](#need-help) and [how to contribute](#want-to-contribute) to the ongoing development of OpenThread
 
+# OpenThread Features
 
-# Getting started #
+OpenThread implements all features defined in the [Thread 1.1.1 specification](http://threadgroup.org/technology/ourtechnology#specifications), including all Thread networking layers (IPv6, 6LoWPAN, IEEE 802.15.4 with MAC security, Mesh Link Establishment, Mesh Routing) and device roles.
 
-The easiest way to get started is to run the CLI example in
-`/examples/apps/cli`. See the
-[CLI example README](examples/apps/cli/README.md)
-for more details.
+Other OpenThread-specific features and enhancements include:
 
+* Network authentication with Commissioning
+* IPv6 configuration and raw data interface
+* UDP sockets
+* CoAP client and server
+* DHCPv6 client and server
+* DNSv6 client
+* Command Line Interface (CLI)
+* Network Co-Processor (NCP) support - off-loading Thread functions to a SoC
+* Spinel - general purpose NCP protocol
+* `wpantund` - user-space NCP network interface driver/daemon
+* Sniffer support via NCP Spinel nodes
 
-## What's included ##
+### What's coming?
 
-In the repo you'll find the following directories and files:
+The development of OpenThread is ongoing to regularly provide additional features not available in the standard. Currently, we are working on:
 
-File/Folder   | Provides
---------------|----------------------------------------------------------------
-`doc`         | Code and protocol docs
-`etc`         | Configuration files for other build systems (e.g. Visual Studio)
-`examples`    | Sample applications and platforms demonstrating OpenThread
-`include`     | Public API header files
-`src`         | Core implementation of the Thread standard and related add-ons
-`tests`       | Unit and Thread conformance tests
-`third_party` | Third-party code used by OpenThread
-`tools`       | Helpful utilities related to the OpenThread project
+* Host-side Border Agent support
+
+# Who supports OpenThread?
+
+Led by Nest, the following companies are contributing to the ongoing development of OpenThread:
+
+[![ARM][arm-logo]][arm-link][![Atmel][atmel-logo]][atmel-link][![Dialog][dialog-logo]][dialog-link][![Microsoft][ms-logo]][ms-link][![Nest][nest-logo]][nest-link][![Nordic][nordic-logo]][nordic-link][![Qualcomm][qc-logo]][qc-link][![Texas Instruments][ti-logo]][ti-link]
```

Comment:
nit: would be nice if these logos were evenly distributed in their layout like they were in the single image.